README.md

```diff
@@ -57,15 +57,17 @@ ParadeDB is currently in Public Beta. Star and watch this repository to get noti
 
 To get started, please visit our [documentation](https://docs.paradedb.com).
 
-## Creating a ParadeDB Instance
+## Deploying ParadeDB
 
-### ParadeDB Cloud
+ParadeDB and its extensions, `pg_analytics` and `pg_search`, are available as commercial software for installation on self-hosted Postgres deployment, and via Docker and Kubernetes as standalone images. For more information, including enterprise features and support, please [contact us by email](mailto:sales@paradedb.com) or [fill out our contact form](https://paradedb.typeform.com/to/jHkLmIzx).
```

Comment:
Just give people one way to reach out, don't confuse them by providing two options.

Reply:
Fair, done